net/MapViewController.m

```diff
@@ -103,7 +105,7 @@ -(void)viewWillAppear:(BOOL)animated
         self.currentUser = [CurrentItems sharedItems].user;
     }
     
-    if(self.currentMarker!=nil)
+    if([self isMarkerSelected])
```

Comment:
It's just a style. But,to tell the truth, it doesn't really matter for me, which of two to use.
Besides as usual, when I use dot notation, I'll use it explicitly 
if (self.isMarkerSelected == YES)